### ISSUE TITLE
Stop words not used by the engine

### ISSUE THREAD

**Issue by clemp6r** [PredictionIO/template-scala-parallel-textclassification#4]
The stop words are read from the event DB but they are never used by the engine (unless I misunderstood something...).

**Comment by marcovivero** [comment]
You're right they're not currently being used in the engine. You can add these in easily as they're an element of `TrainingData` case class. Just modify the `TFHasher` class to take the stopwords as an input parameter, and apply the appropriate filter here:

https://github.com/apache/incubator-predictionio-template-text-classifier/blob/master/src/main/scala/Preparator.scala#L68

You can subsequently pass in the stopwords here:

https://github.com/apache/incubator-predictionio-template-text-classifier/blob/master/src/main/scala/Preparator.scala#L29

**Comment by nlauchande** [comment]
Hi guys i have sent a PR : https://github.com/apache/incubator-predictionio-template-text-classifier/pull/9 with a tentative implementation of the described solution.

**Comment by nlauchande** [comment]
@marcovivero  can you take a look at the PR : https://github.com/apache/incubator-predictionio-template-text-classifier/pull/9 , i have been using this fix in production for a while. Wanted to get it merged upstream on the template, available to make changes upon suggestions.
